Gemfile

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
-gemspec
\ No newline at end of file
+gemspec
+
+gem 'rack', '< 2.0'
```

Comment:
This too should be in the gemspec if the gem it self is not compatible with rack 2+

Reply:
It is compatible, I just wanted to get to green. 

I’ll do the right thing: tell Bundler about which Ruby I’m on. 

That way, the package version resolution will take supported Ruby version into account.

Reply:
There.